dart/lib/src/sentry_client.dart

```diff
@@ -155,49 +153,50 @@ class SentryClient {
 
   /// Reports the [template]
   Future<SentryId> captureMessage(
-    String formatted, {
-    SentryLevel level,
-    String template,
-    List<dynamic> params,
-    Scope scope,
+    String? formatted, {
+    SentryLevel level = SentryLevel.info,
```

Comment:
```suggestion
    SentryLevel level,
```

please double check all the `SentryLevel level = SentryLevel.info`

Reply:
SentryLevel is a required field (because it is not nullable) If it doesn't have a default value  which would mean that anyone who is using the Hub directly would always need to specify a SentryLevel. Should I still make the change?

Reply:
I will resolve this one in favor of https://github.com/getsentry/sentry-dart/pull/298#discussion_r580836701
so depending on what we decide, we change for all of them